extensions/VisualEditor/wikia/VisualEditor.php

```diff
@@ -24,6 +24,18 @@
 			'ext.visualEditor.track'
 		),
 		'position' => 'top'
-	)
+	),
+	'ext.visualEditor.wikiaViewPageTarget' => $wgVisualEditorWikiaResourceTemplate + array(
```

Comment:
Good call. I just aligned our naming scheme more closely with wmf. They do some modules like, "ext.visualEditor.mediawiki.mobile" and other like, "ext.visualEditor.mobileViewTarget". I committed some placeholders to establish a naming pattern.